components/provisioner/internal/gardener/annotations.go

```diff
@@ -21,8 +21,8 @@ func (s KymaInstallationState) String() string {
 }
 
 const (
-	operationIdAnnotation string = "compass.provisioner.kyma-project.io/operation-id"
-	runtimeIdAnnotation   string = "compass.provisioner.kyma-project.io/runtime-id"
+	operationIdAnnotation string = "kcp.provisioner.kyma-project.io/operation-id"
+	runtimeIdAnnotation   string = "kcp.provisioner.kyma-project.io/runtime-id"
```

Comment:
I am not sure if we can just change these annotations as they are used by other teams. I suggest introducing the second set of annotations starting with `kcp` and inform others about the change to give them time to switch.